README.reproducible.md

```diff
@@ -0,0 +1,148 @@
+[fetch and build inputs][fetch and build inputs].
+
+Note that you may (read will) need to grab the dependency sources manually. See
+[seed cache][seed cache]. Just replace bitcoin in the instructions with
+BitcoinArmory or whatever your Armory source tree is named.
+
+##Linux
+
+###Reproducible
+
+For Linux reproducible builds, we have used the
+[Debian Reproducible Build Toolchain][Debian Toolchain]. We have found the
+toolchain to be better suited to producing reproducible Debian packages simply
+due to the fact that it was designed specifically with Debian packages in mind.
+
+For more on producing a reproducible Debian package of Armory, read
+[dpkgfiles/make\_deb\_package.README](dpkgfiles/make_deb_package.README)
+
+###Non-reproducible
+
+If you don't want to build for Linux reproducibly, or you are building for a
+system that does not support deb or rpm packages, you can just build using the
+Autotools-based system directly. This has only been tested when building on
+Linux.
+
+    cd path-to-armory-source
+    ./autogen.sh
+    ./configure # don't need to specify host if building on same architecture as host
+    make
+    make install # optional
+
+Specify `--host x86_64-linux-gnu` or `--host i686-linux-gnu` if you are
+building for 64-bit from a 32-bit machine or building 32-bit from a 64-bit
+machine respectively.
+
+If you don't run `make install`, you can just run `python ArmoryQt.py` from the
+Armory source tree. If you run `make install`, you should have `armory` on your
+path. So you can just run `armory`.
+
+###Fedora Package (RPM)
+
+RPM packages are not officially supported. They are a convenience to users.
+Read [rpmfiles/armory.spec.README](rpmfiles/armory.spec.README) for more info.
+
+##OS X
+
+###Reproducible
+
+The OS X reproducible build system uses Gitian. Refer to the Gitian
+instructions and use gitian-descriptors/bitcoin-armory-osx.yml as the Gitian
+descriptor.
+
+###Non-reproducible
+
+You can use the Autotools-based build system to build Armory.app. This has been
+tested on OS X and Linux as the build machines.
+
+For OS X:
+
+    cd path-to-armory-source
+    ./configure
+    make Armory.app
+
+For Linux:
+
+    # install cctools-port, TODO: add instructions for installing cctools-port
+    echo -e '#!/bin/sh\nexit 0' > /usr/local/bin/x86_64-apple-darwin11-dsymutil
+    mv /usr/local/bin/x86_64-apple-darwin11-ld /usr/bin/
+    cd path-to-armory-source
+    ./autogen.sh
+    ./configure --host=x86_64-apple-darwin11 \
+                CC="clang -target x86_64-apple-darwin11 -mmacosx-version-min=10.7 --sysroot /usr/share/MacOSX10.9.sdk -mlinker-version=241.9" \
+                CXX="clang++ -target x86_64-apple-darwin11 -mmacosx-version-min=10.7 --sysroot /usr/share/MacOSX10.9.sdk -mlinker-version=241.9"
+    make Armory.app
+
+In either case, you should be left with an Armory.app in osxbuild that will run
+on OS X 10.7+.
+
+Alternatively, you can just run `make` if you don't want to build Armory.app.
```

Comment:
Have you tried this? I was under the impression that you needed to build the whole app. I suppose it could just be a byproduct of how things currently work, though, and not how they can work in general.